code/patrick/lab13v1.py

```diff
@@ -1,50 +1,180 @@
 class Player:
-    def _init_(self, name, token):
-       self.name = name
-       self.token = token # X or O
+    def _init_(self, token):
+       self.name = " "
+  
 
-class Game:
-    def _init_ (self, board):
-        self.board_1 = self
+    def player_name(self):
```

Comment:
there's nothing to say you couldn't set the name with a method, but if the `__init__` takes player and token params, it makes more sense that you would instantiate a player with those 2 strings

Reply:
Deleted the name method. Asked for name input in Main(). Added self.token under __init__.